charts/newrelic-infrastructure/README.md

```diff
@@ -1,29 +1,102 @@
+[helm-charts repository](https://github.com/newrelic/helm-charts) or directly from this repository by adding this Helm repository:
+
+```shell
+helm repo add nri-kube-events https://newrelic.github.io/nri-kube-events
+helm upgrade --install nri-kube-events/nri-kube-events -f your-custom-values.yaml
+```
+
 ## Source Code
 
 * <https://github.com/newrelic/nri-kubernetes/>
-* <https://github.com/newrelic/helm-charts/tree/master/charts/newrelic-infrastructure>
+* <https://github.com/newrelic/nri-kubernetes/tree/master/charts/newrelic-infrastructure>
+* <https://github.com/newrelic/infrastructure-agent/>
 
 ## Requirements
 
-Kubernetes: `>=1.16.0-0`
+| Repository | Name | Version |
+|------------|------|---------|
+| https://helm-charts.newrelic.com | common-library | 1.0.2 |
+
+## Values managed globally
+
+This chart implements the [New Relic's common Helm library](https://github.com/newrelic/helm-charts/tree/master/library/common-library) which
+means that is has a seamless UX between things that are configurable across different Helm charts. So there are behaviours that could be
+changed globally if you install this chart from `nri-bundle` or your own umbrella chart.
+
+A really broad list of global managed values are `affinity`, `nodeSelector`, `tolerations`, `proxy` and many more.
+
+For more information go to the [user's guide of the common library](https://github.com/newrelic/helm-charts/blob/master/library/common-library/README.md)
+
+## Chart particularities
+
+### Low data mode
+There are two mechanisms to reduce the amount of data that this integration sends to New Relic. See this snippet from the `values.yaml` file:
+```yaml
+common:
+  config:
+    interval: 15s
+
+lowDataMode: false
+```
+
+The `lowDataMode` toggle is the simplest way amd setting it to `true` changes the default scrape interval from 15 seconds (the default) to 30 seconds.
+
+If you need for some reason to fine tune the amount of seconds you can use `common.config.interval` directly. If you take a look to the `values.yaml`
+file, the value there is `nil`. If set any value there, the `lowDataMode` toggle is ignored and this value is issued.
+
+Setting this interval above 40 seconds can make you experience issues with the Kubernetes Cluster Explorer so this chart limits setting the interval
+inside the range of 10 to 40 seconds.
+
+### Affinities and tolerations
+
+The New Relic common library allows to set affinities, tolerations and nodeSelectors in globally with the resto of the integrations and products. This
+integration in particular has affinities and tolerations set to be able to schedule pods in nodes that are tainted as master nodes and to schedule a
+pod near the KSM to reduce the inter-node traffic.
+
+Take a look to the [`values.yaml`](values.yaml) so see how to configure them if you are having problems scheduling pods where you want to.
```

Comment:
I do not like that much this sentence, can we improve it somehow?

Reply:
I rewrite a part of it in [#0fb8df2](https://github.com/newrelic/nri-kubernetes/pull/409/commits/0fb8df26c7f339111945dbe2bd0c2af63943cbc0)